aquarius/events/events_monitor.py

```diff
@@ -27,6 +27,7 @@
     TokenURIUpdatedProcessor,
 )
 from aquarius.events.purgatory import Purgatory
+from aquarius.events.veAllocate import VeAllocate
```

Comment:
please use Python standard `snake_case` when naming files and variables, `CamelCase` only for classes